### PR TITLE
Add one more closed to do issue pattern and fix violations

### PR DESCRIPTION
**Pull request by strkkk**
Related to #9256 

I found one more pattern for to do issues, it is like `till #<issue number>`

1. This pattern is added to job for checking source code
2. Violations fixed for closed items
3. Second commit - found bug in checkstyle check for checking whitespaces after comments in sh files. Added a new check for it and fixed violations. May be new PR is needed for this change?

### CODE REVIEW

**Comment by strkkk** [comment]
@romani I need some help here
https://github.com/checkstyle/checkstyle/blob/b51e96403cf844fa805113e4570f33bc36314388/config/intellij-idea-inspections.xml#L2755-L2761

#5949 is closed, but I am not sure what should be done to resolve this todo item.
What is a new name for this check? Or name is same and it is enough to just enable it?

**Comment by strkkk** [comment]
Travis failure is expected and caused by unresolved item in https://github.com/checkstyle/checkstyle/pull/9276#issuecomment-778754488

File: config/default_sonar_profile.xml [hunk 1]
```diff
@@ -9,45 +9,36 @@ e.g. - provided link corresponds to rule S3776
...
   <rules>
-    <!-- till #4912 -->
-    <!--
     <rule>
       <repositoryKey>common-java</repositoryKey>
       <key>DuplicatedBlocks</key>
       <priority>MAJOR</priority>
       <parameters/>
     </rule>
-    -->
```
Comment (strkkk) [inline config/default_sonar_profile.xml:20]: @romani I also not so sure about these uncommented sonar items. Are they needed?

Comment (romani) [reply config/default_sonar_profile.xml:20]: we do not use this file , we do not use sonar in docker - https://github.com/checkstyle/checkstyle/issues/4912

**Review by romani — APPROVED** [review]
ok to merge
 
> What is a new name for this check? Or name is same and it is enough to just enable it?

I do not remember new name, try to activate it. 
we can always find this inspection rule in IDEA UI , they usually reasonable by name. To new name we can export config in xml and compare files to detect new rule.

**Comment by strkkk** [comment]
@romani I enabled this rule (it has same name) and fixed violations:
1. Rule max length extended to 65 (because longest test name `XpathRegressionUnnecessarySemicolonAfterTypeMemberDeclarationTest` has 65 chars in it)
2. 3 other violations were suppressed.
3. There was a mention of ticket https://youtrack.jetbrains.com/issue/IDEA-187210, it is fixed, so I removed it.

Please check changes, may be you have some comments for it

File: config/default_sonar_profile.xml (hunk 1 above)
Comment (strkkk) [reply config/default_sonar_profile.xml:20]: done https://github.com/checkstyle/checkstyle/pull/9284

**Review by romani — APPROVED** [review]
ok to merge if CI is green

**Comment by romani** [comment]
@strkkk , please resolve conflict.

**Comment by strkkk** [comment]
Conflict is resolved.
@pbludov please take a look

**Review by pbludov — APPROVED** [review]

**Pull request merged**